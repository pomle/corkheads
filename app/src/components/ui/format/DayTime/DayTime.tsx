import React from "react";
import { Moment } from "moment";

const FORMAT = "YYYY-MM-DD HH:mm";

interface DayTimeProps {
  date: Moment;
}

const DayTime: React.FC<DayTimeProps> = ({ date }) => {
  return <>{date.format(FORMAT)}</>;
};

export default DayTime;
