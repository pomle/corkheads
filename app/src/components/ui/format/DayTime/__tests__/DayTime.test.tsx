import React from "react";
/* eslint import/no-extraneous-dependencies: ["error", {"peerDependencies": true}] */
import renderer from "react-test-renderer";
import moment from "moment";
import DayTime from "../DayTime";

describe("DayTime component", () => {
  const VALID_MOMENT = moment("2019-09-12T12:42:13.423Z");

  it("shows the date and time of date when valid", () => {
    const component = renderer.create(<DayTime date={VALID_MOMENT} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
