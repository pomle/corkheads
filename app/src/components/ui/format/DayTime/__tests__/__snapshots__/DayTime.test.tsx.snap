// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DayTime component shows the date and time of date when valid 1`] = `"2019-09-12 12:42"`;
